### DISCUSSION TITLE
Any i18n plan?

### DISCUSSION THREAD

**Discussion by LenKiMo** [simple-login/app#746] [Feature Requests]
I just recently started using SimpleLogin and it has really provided me with a great experience.

However, I noticed that the website and app are currently only available in English. I wonder if SimpleLogin has any translation and localization platform, or if the software has a pre-defined localization structure to facilitate translation to different languages in the future?

**Comment by nguyenkims** [comment]
Hi the website is actually also available in French https://simplelogin.io/fr/

We have started localisation project some times ago but haven't had time to continue it ... Once completed, anyone can contribute and improve the translations.
